shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -138,7 +138,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
             result.setAccessibilityFocused(mA11yFocusedObject.id == virtualViewId);
 
         if (object.hasFlag(Flag.IS_TEXT_FIELD)) {
-            result.setPassword(object.hasFlag(Flag.IS_PASSWORD);
+            result.setPassword(object.hasFlag(Flag.IS_PASSWORD));
```

Comment:
this is already fixed in the tip of tree

Reply:
Fixed